openmp/libomptarget/src/PluginManager.cpp

```diff
@@ -23,11 +23,15 @@ PluginManager *PM;
 
 // List of all plugins that can support offloading.
 static const char *RTLNames[] = {
+#ifdef ENABLED_OFFLOAD_PLUGINS
```

Comment:
`ENABLED_OFFLOAD_PLUGINS` now is already defined right?

Reply:
Right. I can get rid of the alternative here.